### DISCUSSION TITLE
Releases and SimRel

### DISCUSSION THREAD

**Discussion by merks** [eclipse/tm4e#539] [Q&A]
If I look here that tm4e's contribution to SimRel

https://git.eclipse.org/c/simrel/org.eclipse.simrel.build.git/tree/tm4e.aggrcon

I see this repository is contributed:

https://download.eclipse.org/tm4e/releases/0.6.2/

But I see these newer release records

- https://projects.eclipse.org/projects/technology.tm4e/releases/0.7.1
- https://projects.eclipse.org/projects/technology.tm4e/releases/0.7.0
- https://projects.eclipse.org/projects/technology.tm4e/releases/0.6.3

I also see corresponding repositories here:

https://download.eclipse.org/justj/?file=tm4e/releases

Is tm4e still participating and if so, should the contribution be updated?

**Comment by mickaelistria** [comment]
I can't tell for other contributors (mostly @sebthom ), but FWIW, as my priorities have been re-targeted not so long ago, dealing with SimRel (and EPP) is something I personally cannot dedicate much time to.
However, as TM4E is an important part of Wild Web Developer, which is itself an important part of m2e pom editor, and TM4E receives regular maintenance, I think it's definitely worth trying to keep it up to date for the overall quality of SimRel and EPP.

**Comment by merks** [comment]
Yes, I figured since there were releases with bug fixes, it makes sense to update the contribution, unless there is a reason not to do that.

**Comment by sebthom** [comment]
What are the TODOs here? I am not familiar with the SimRel process.

**Comment by merks** [comment]
There's documentation here, including an automated Oomph setup:

https://ci.eclipse.org/simrel/

There's more detailed steps here:

https://wiki.eclipse.org/Simrel/Contributing_to_Simrel_Aggregation_Build

So it's just periodically maintaining the repository information in this resource:

- https://git.eclipse.org/c/simrel/org.eclipse.simrel.build.git/tree/tm4e.aggrcon
- https://git.eclipse.org/c/simrel/org.eclipse.simrel.build.git/log/tm4e.aggrcon
